docs/apache-airflow-providers-celery/celery_executor.rst

```diff
@@ -187,10 +187,10 @@ Two databases are also available:
 - QueueBroker
 - ResultBackend
 
-During this process, two 2 process are created:
+During this process, two processes are created:
 
-- LocalTaskJobProcess - It logic is described by LocalTaskJob. It is monitoring RawTaskProcess. New processes are started using TaskRunner.
-- RawTaskProcess - It is process with the user code e.g. :meth:`~airflow.models.baseoperator.BaseOperator.execute`.
+- LocalTaskJobProcess - It's logic is described by LocalTaskJob. It is monitoring RawTaskProcess. New processes are started using TaskRunner.
+- RawTaskProcess - It is a process with the user code e.g. :meth:`~airflow.models.baseoperator.BaseOperator.execute`.
```

Comment:
Not sure about this one:
```suggestion
- RawTaskProcess - It is a process within the user code e.g. :meth:`~airflow.models.baseoperator.BaseOperator.execute`.
```

Reply:
I guess this means that the process contains the user code, based on this diagram: https://airflow.apache.org/docs/apache-airflow-providers-celery/stable/_images/run_task_on_celery_executor.png